lib/Runtime/Library/WasmLibrary.cpp

```diff
@@ -214,7 +214,7 @@ namespace Js
             funcObj->SetEnvironment(frameDisplay);
             localModuleFunctions[i] = funcObj;
 
-            if (PHASE_ON(WasmDeferredPhase, funcObj->GetFunctionBody()))
+            if (!PHASE_OFF(WasmDeferredPhase, funcObj->GetFunctionBody()))
```

Comment:
not even a nit. i understand some people typically try to avoid double negations. In this particular case **false** and **true** blocks pretty simple anyways, so it might make sense to get rid of double negation and switch the **false** arm with **true**

```cpp
            if (PHASE_OFF(WasmDeferredPhase, funcObj->GetFunctionBody()))
            {
   WasmFunctionGenerateBytecode(funcObj, !PHASE_ON(WasmLazyTrapPhase, funcObj->GetFunctionBody()));

            }
            else
            {
                funcObj->GetDynamicType()->SetEntryPoint(WasmLibrary::WasmDeferredParseExternalThunk);
                entypointInfo->jsMethod = WasmLibrary::WasmDeferredParseInternalThunk;             
            }
```